automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -272,6 +272,24 @@ groups:
       description: "The rate of {{ $value }} new blocks observed by archive postgres instances is low on network {{ $labels.testnet }}."
       runbook: "https://www.notion.so/minaprotocol/Archive-Node-Metrics-9edf9c51dd344f1fbf6722082a2e2465"
 
+  - alert: NodeRestarted
+    expr: count by (testnet) (Coda_Runtime_process_uptime_ms_total{testnet=~"mainnet|devnet2|snappnet"} < 360000) > 0
+    labels:
+      testnet: "{{ $labels.testnet }}"
+      severity: warning
+    annotations:
+      summary: "At least one of the nodes on {{ $lables.testnet }} restarted"
+      description: "{{ $value }} nodes on {{ $labels.testnet }} restarted"
+
+  - alert: MultipleNodeRestarted
+    expr: count by (testnet) (Coda_Runtime_process_uptime_ms_total{testnet=~"mainnet|devnet2|snappnet"} < 600000) > 2
+    labels:
+      testnet: "{{ $labels.testnet }}"
+      severity: critical
```

Comment:
@mrmr1993  did you mean to use `snappnet` in the `testnet` label here to generate pagerduty alerts?  The variable `rule_filter` currently includes snappnet as well and you can see those in grafana alert notifications. If you wanted pagerduty alert for this we'll have to update `pagerduty_alert_filter` variable (but this will apply to all the alerts). 

I made this PR to show mainnet|devnet warnings and other testnet alerts in a slack channel <https://github.com/MinaProtocol/mina/pull/8707>. If that is not sufficient then I think we could add entries in `testnet-alert-receivers.yml.tpl` for matching specific alerts from specific testnets and sending them to specific receivers (pagerduty or slack channel)

Reply:
@ghost-not-in-the-shell could you remove the hardcoded testnet names and use `${rule_filter}`?

Reply:
I already did that.